src/binary/binaryFetcher/bundleDownloader.ts

```diff
@@ -65,7 +67,15 @@ function createBundleDirectory(bundleDirectory: string): Promise<void> {
 
 async function getCurrentVersion(): Promise<string> {
   const versionUrl = getUpdateVersionFileUrl();
-  return downloadFileToStr(versionUrl);
+  const version = await downloadFileToStr(versionUrl);
+  validateVersion(version);
+  return version;
+}
+
+function validateVersion(version: string): void {
```

Comment:
rename to `assertValidVersion` so it will be easier to understand that this function throws an error